src/webgpu/api/operation/rendering/draw.spec.ts

```diff
@@ -645,3 +645,13 @@ g.test('vertex_attributes,formats')
   `
   )
   .unimplemented();
+
+g.test(`largeish_buffer`)
+  .desc(
+    `
+    Test a very large range of buffer is bound to different slot.
```

Comment:
nit: what do you mean by "different" slot?

probably we should test both slots that are/aren't used by the current pipeline.

Reply:
Still need to update wording to be clearer about what is tested here.

Reply:
Done.